jobs/config.json

```diff
@@ -13086,6 +13086,27 @@
       "sig-scheduling"
     ]
   },
+  "pull-release-null-e2e": {
+    "args": [
+      "--build=bazel",
+      "--cluster=",
+      "--env-file=jobs/platform/gce.env",
+      "--env-file=jobs/env/pull-kubernetes-e2e.env",
+      "--extract=local",
+      "--gcp-node-image=gci",
+      "--gcp-project=k8s-jkns-pr-gce-etcd3",
```

Comment:
just use boskos pool

Reply:
Ok, this is copied from e2e-gce presubmit, will remove these

Reply:
done